contribute.md

```diff
@@ -0,0 +1,46 @@
+Contributing
+
+Contributions are always welcome, no matter how large or small! Before contributing, please read the code of conduct.
+
+Not sure where to start?
+When you feel ready to jump into the source code, a good place to start is to look for issues tagged with help wanted and/or good first issue.
+
+Setup
+install python v3
+install pip
+intall tensorflow v2
+
+setup pytest as test runner in your IDE
+setup crlf in git config
+
+
+How to run tests
+execute from command line command pytest
+
+Pull Request Checklist
+
+Fork the repository to your GitHub Account.
+Clone forked repo to your local machine
+Create your feature branch
+Make code/documantation changes
+Create pull request
+
+Before sending your pull requests, make sure you followed this list.
+
+Read contributing guidelines.
+Read Code of Conduct.
```

Comment:
Where should the Code of Conduct be placed? Should it be a separate file, or can one add it just here?

Reply:
It is usual separate file, and threre is a link to code of conduct in readmi

Reply:
Code of Conduct is optional, it is actual for project with contributors community